object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
```

Comment:
I'm not sure I understand what you're shooting for - this behavior should be intrinsic to the object browser pattern